shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -116,4 +116,23 @@ - (void)viewDidChangeBackingProperties {
   [_reshapeListener viewDidReshape:self];
 }
 
+#pragma mark - NSAccessibility overrides
+
+- (BOOL)isAccessibilityElement {
+  return self.accessibilityChildren != nil && [self.accessibilityChildren count] != 0;
+}
+
+- (NSAccessibilityRole)accessibilityRole {
+  return NSAccessibilityGroupRole;
+}
+
+- (NSString*)accessibilityLabel {
+  NSString* applicationName =
+      [NSBundle.mainBundle objectForInfoDictionaryKey:@"CFBundleDisplayName"];
+  if (!applicationName) {
+    applicationName = [NSBundle.mainBundle objectForInfoDictionaryKey:@"CFBundleName"];
+  }
+  return applicationName;
```

Comment:
Can someone ever create multiple FlutterViews in a single macOS application?

e.g. in an "add to app" scenario

Reply:
interesting I never thought about this, not sure how viable it is. but maybe we need an API to customize this. That can be a separate PR though

Reply:
Maybe jus tmake it a property on the view that can be set?